source/_includes/common-tasks/backups.md

```diff
@@ -12,6 +12,16 @@ A full backup includes the following directories:
 
 A partial backup consists of any number of the above default directories and installed add-ons.
 
+### Preparing for a backup
+
+1. Before creating a backup, check if you can reduce the size of the backup.
+   - Check if your configuration directory contains a large database file (`home-assistant_v2.db`). 
+   - See the [`recorder`](/integrations/recorder/) integration page for options to keep your database data down to a size that won't cause issues. 
+   - Note the keep days, purge interval, and include/exclude options.
+   - If you have add-ons installed that you no longer use, delete those add-ons. Some add-ons require quite a bit of space.
+2. Delete all old and unneeded backups.
```

Comment:
Deleting old backups are not really doing anything for the next backup as those are not included.
So having that nested under "preparing for a backup" does not feel right.

Maybe we should have a "Free up used space" section where this can be instead?